crates/rpc/rpc/src/eth/api/fees.rs

```diff
@@ -34,14 +34,12 @@ where
 
     /// Returns a suggestion for a gas price for blob transactions.
     pub(crate) async fn blob_gas_price(&self) -> EthResult<U256> {
-        let header = self.block(BlockNumberOrTag::Latest).await?;
-        Ok(U256::from(
-            header
-                .unwrap_or_default()
-                .excess_blob_gas
-                .ok_or(EthApiError::ExcessBlobGasNotSet)
-                .map(calc_blob_gasprice)?
-        ))
+        self.block(BlockNumberOrTag::Latest)
+            .await?
+            .unwrap_or_default()
```

Comment:
style: this should be `and_then`, so we don't unnecessarily create an empty header, even though `latest` should always be available